debian/jellyfin.init

```diff
@@ -9,13 +9,14 @@
 ### END INIT INFO
 
 # Carry out specific functions when asked to by the system
-pid=`ps -fA|grep dotnet|grep JellyfinServer|awk '{print $2}'| tr -d '\n'`
+pid=`ps -fA|grep dotnet|grep jellyfin|awk '{print $2}'| tr -d '\n'`
```

Comment:
This should probably be grepping for `$JELLYFIN_EXE` (and config could be sourced earlier)

Reply:
That "original script" was mine I think... It was hacked in like 5 minutes, not something to be kept intact :)